x-pack/plugins/observability_solution/synthetics/server/routes/overview_trends/fetch_trends.ts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SearchRequest } from '@elastic/elasticsearch/lib/api/types';
+import { SyntheticsEsClient } from '../../lib';
+
+export function fetchTrends(configId: string, locationIds: string[], esClient: SyntheticsEsClient) {
+  const query: SearchRequest = {
+    size: 0,
+    query: {
+      bool: {
+        filter: [
+          {
+            bool: {
+              filter: [
+                {
+                  exists: {
+                    field: 'summary',
+                  },
+                },
+                {
+                  bool: {
+                    should: [
+                      {
+                        bool: {
+                          should: [
+                            {
+                              match: {
+                                'summary.final_attempt': true,
+                              },
+                            },
+                          ],
+                          minimum_should_match: 1,
+                        },
+                      },
+                      {
+                        bool: {
+                          must_not: {
+                            bool: {
+                              should: [
+                                {
+                                  exists: {
+                                    field: 'summary.final_attempt',
+                                  },
+                                },
+                              ],
+                              minimum_should_match: 1,
+                            },
+                          },
+                        },
+                      },
+                    ],
+                    minimum_should_match: 1,
+                  },
+                },
+              ],
+            },
+          },
+          {
+            bool: {
+              must_not: {
+                exists: {
+                  field: 'run_once',
+                },
+              },
+            },
+          },
+          {
+            terms: {
+              'observer.name': locationIds,
+            },
+          },
+          {
+            term: {
+              config_id: configId,
+            },
+          },
+          {
+            range: {
+              '@timestamp': {
+                gte: 'now-9h',
+                lte: 'now',
+              },
+            },
```

Comment:
what is the logic behind `now-9h`?

Reply:
The previous query [computed this filter](https://github.com/elastic/kibana/pull/187092/files#diff-8a8f2876f220e753d3536856266f4d9b8054474f67b9d6ea63ef4a12fd8d0bb5L23) and I likely picked this as a happy default while developing, and I hadn't cleaned up the query before the first review happened 😄.